### PR TITLE
122 updating purchase module

### PR DESCRIPTION
**Pull request by Asadupramod012**
updated purchase module

### CODE REVIEW

File: apps/purchase/serializers.py [hunk 1]
```diff
@@ -20,6 +20,7 @@ class PurchaseOrdersSerializer(serializers.ModelSerializer):
     payment_term = ModVendorPaymentTermsSerializer(source='payment_term_id',read_only=True)
     purchase_type = PurchaseTypesSerializer(source='purchase_type_id',read_only=True)
     ledger_account = ModLedgerAccountsSerializers(source='ledger_account_id',read_only=True)
+    #order_status = ModOrderStatusesSerializers(source='order_status_id',read_only=True)
```
Comment (QodeFarm) [inline apps/purchase/serializers.py:23]: Why this is commented?

File: apps/purchase/serializers.py [hunk 2]
```diff
@@ -37,21 +38,64 @@ class PurchaseorderItemsSerializer(serializers.ModelSerializer):
...
 
+class PurchaseInvoiceOrdersSerializer(serializers.ModelSerializer):
+    gst_type = ModGstTypesSerializer(source='gst_type_id',read_only=True)
+    vendor = ModVendorSerializer(source='vendor_id',read_only=True)
+    vendor_agent = ModVendorAgentSerializer(source='vendor_agent_id',read_only=True)
+    vendor_address = VendorAddressSerializer(source='vendor_address_id',read_only=True)
+    payment_term = ModVendorPaymentTermsSerializer(source='payment_term_id',read_only=True)
+    purchase_type = PurchaseTypesSerializer(source='purchase_type_id',read_only=True)
+    ledger_account = ModLedgerAccountsSerializers(source='ledger_account_id',read_only=True)
+    # order_status = ModOrderStatusesSerializers(source='order_status_id',read_only=True)
```
Comment (QodeFarm) [inline apps/purchase/serializers.py:55]: Why this is commented?

File: apps/purchase/serializers.py [hunk 3]
```diff
@@ -37,21 +38,64 @@ class PurchaseorderItemsSerializer(serializers.ModelSerializer):
...
+        fields = ['purchase_return_id', 'return_date', 'return_no']
+
+class PurchaseReturnOrdersSerializer(serializers.ModelSerializer):
+    purchase_type = PurchaseTypesSerializer(source='purchase_type_id',read_only=True)
+    gst_type = ModGstTypesSerializer(source='gst_type_id',read_only=True)
+    vendor = ModVendorSerializer(source='vendor_id',read_only=True)
+    vendor_agent = ModVendorAgentSerializer(source='vendor_agent_id',read_only=True)
+    vendor_address = VendorAddressSerializer(source='vendor_address_id',read_only=True)
+    payment_term = ModVendorPaymentTermsSerializer(source='payment_term_id',read_only=True)
+    # order_status = ModOrderStatusesSerializers(source='order_status_id',read_only=True)
```
Comment (QodeFarm) [inline apps/purchase/serializers.py:85]: Why this is commented?

File: apps/purchase/urls.py [hunk 4]
```diff
@@ -6,9 +6,12 @@
 router = routers.DefaultRouter()
 router.register(r'purchase_orders', PurchaseOrdersViewSet)
 router.register(r'purchaseorder_items', PurchaseorderItemsViewSet)
```
Comment (QodeFarm) [inline apps/purchase/urls.py:8]: change this purchaseorder_items to purchase_order_items

Comment (Asadupramod012) [reply apps/purchase/urls.py]: Chnaged to purchase_order_items

**Pull request merged**